rslib/src/media/files.rs

```diff
@@ -550,4 +564,20 @@ mod test {
             Cow::<str>::Owned(format!("{}_", " ".repeat(MAX_MEDIA_FILENAME_LENGTH - 2)))
         );
     }
+    #[test]
+    fn safe_rename_moves_file() {
```

Comment:
This test is not really confirming the function works across disks. But I'm not sure how difficult it is to test that.